t/14-lb_req_dc_rr.t

```diff
@@ -454,3 +454,30 @@ local_dc: dc1
 3. 127.0.0.3
 --- no_error_log
 [error]
+
+
+
+=== TEST 9: lb_req_dc_rr with down node no error
+--- http_config eval: $::HttpConfig
+--- config
+    location /t {
+        content_by_lua_block {
+            local req_dc_rr = require 'resty.cassandra.policies.lb.req_dc_rr'
+
+            local peers = {
+                {host = '10.0.0.1', err = 'connection refused'},
+                {host = '10.0.0.2', data_center = 'dc1'},
+                {host = '10.0.0.3', data_center = 'dc1'},
+            }
+
+            local lb = req_dc_rr.new('dc1')
+
+            lb:init(peers)
+        }
+    }
+--- request
+GET /t
+--- error_code: 200
+--- error_log
+[lua-cassandra] peer 10.0.0.1 data_center field must be a string
```

Comment:
This is not the proper test for this assertion. Besides, this test will go away once `.err` isn't logged anymore.